toothpick-runtime/src/main/java/toothpick/Toothpick.java

```diff
@@ -28,6 +28,18 @@ private Toothpick() {
     throw new RuntimeException("Constructor can't be invoked even via reflection.");
   }
 
+  /**
+   * Find scope by this {@code name} in current scopes tree.
+   * @return true if scope exist.
+   */
+  public static boolean hasScope(Object name) {
```

Comment:
can you rename to `isScopeOpen` please ?

And please add unit tests to : `toothpick.ToothpickTest`. We will release it as part of TP 2.1. Thx @terrakok